source/includes/_errors.md

```diff
@@ -14,6 +24,7 @@ Error Code | Meaning
 410 | Gone -- The kitten requested has been removed from our servers
 413 | Request Entity Too Large -- You sent too much data to us
 418 | I'm a teapot
+422 | Unprocessable Entity
```

Comment:
My issue with 400 is that it [relates to syntax](http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html) as opposed to semantics, which is the case with 422. Sending a cart with a quantity of -1 is valid syntax, but its execution results in an invalid entity which is rejected by the server. Sending a 422 is more specific to the type of error we're trying to convey.

My arguments regarding support are that the WebDav RFC has been around for 8 years now which should ensure pretty wide support and, with questionable significance, that this status code has been the default in Rails for validation errors [since 2011](https://github.com/rails/rails/commit/b4359bc7234b61c9a4a104542fa77f63bb84d7e1) (it's also why I used it in the first place, it's just more familiar to me).

Otherwise, it's still a 4xx error, so the client always knows the problem is on their side.